bullet_train-fields/app/javascript/controllers/fields/date_controller.js

```diff
@@ -29,13 +31,13 @@ export default class extends Controller {
     // don't submit the form, unless it originated from the cancel/clear button
     event.preventDefault()
 
-    $(this.fieldTarget).val('')
-    $(this.displayFieldTarget).val('')
+    jquery(this.fieldTarget).val('')
+    jquery(this.displayFieldTarget).val('')
```

Comment:
Can we use `value =` here like in the color picker controller?

```suggestion
    this.fieldTarget.value = ''
    this.displayFieldTarget.value = ''
```